homeassistant/components/xiaomi_miio/air_quality.py

```diff
@@ -32,35 +26,20 @@
 
 PROP_TO_ATTR = {
     "carbon_dioxide_equivalent": ATTR_CO2E,
-    "relative_humidity": ATTR_HUMIDITY,
     "particulate_matter_2_5": ATTR_PM_2_5,
```

Comment:
We don't need to set this attribute since we implement the corresponding property.

Reply:
So where is the relative humidity and temperature now? For as far as I understand it should also create a temperature sensor?

Reply:
Got it, you mean the miio device property is already implemented. :-)